### PR TITLE
tests: Add print checker in conftest

### PR DESCRIPTION
**Pull request by georgebisbas**

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    we call:
+
+    .. code-block:: python
+
+        assert_print_op(add, expected)
+
+    """
+
+    file = StringIO("")
+    if diagnostic is None and target is None:
```
Comment (georgebisbas) [inline tests/conftest.py:44]: need some fixing

**Review by webmiche — APPROVED** [review]
I feel like there is an argument to be had what we want to have in such a file and what not. This specific functionality makes sense to have in a separate file IMO, so LGTM.

**Review by superlopuh — APPROVED** [review]
Neat!

**Comment by webmiche** [comment]
@georgebisbas can you rebase s.t. we can merge this? :)

**Comment by georgebisbas** [comment]
@webmiche rebased

**Comment by webmiche** [comment]
Will merge once the checks are done.

**Pull request merged**